.isort.cfg

```diff
@@ -0,0 +1,13 @@
+[isort]
```

Comment:
Please keep the original config as defined in `setup.cfg`. I'm definitely open to changing it where necessary, but really want to keep this PR as atomic as possible!

Reply:
f183f13